### ISSUE TITLE
Attach explosive not working

### ISSUE THREAD

**Issue by DerZade** [gruppe-adler/TvT_TracksNTraps.gm_weferlingen_summer#68]
I tried to attach explosive to an enemy tank. It instantly aborted (with hint "Aborted"). 

Didn't work for two of my teammates aswell.

**Comment by Dahlgren** [comment]
Didn't work on our own OPFOR tank, just gave hint "Aborted."

**Comment by nomisum** [comment]
https://github.com/gruppe-adler/TvT_TracksNTraps.ruha/blob/9a13946a07a1fd014b6ea1cdce4f94e802725a3e/functions/damageHandling/fn_triggerExplosive.sqf#L29

not sure whats wrong here 🤔 getting closer it should've worked at least

**Comment by DerZade** [comment]
I was directly next to the tank. Both on the side and behind it. So this was definitely to an issue of being too far away.

**Comment by nomisum** [comment]
there was a feedback attaching should be instant, would solve things for sure but ... i fear for this method to be too effective and becoming the main CQB combat mode

**Comment by DerZade** [comment]
@nomisum I agree. Instant would be too powerfull. I had the chance to blow up 3 tanks inside the objective, after mine was destroyed.
Sadly the action did not work so I can't really say if I would have lived after blowing up the first one, but it shows that it is fairly easy to get to a tank within an city.

**Comment by nomisum** [comment]
we could counter this by limiting the explosive amount to one per player, so no one can go rambo alone at least

**Comment by DerZade** [comment]
That's an idea, but I think that is kinda hard to communicate without an Inventory item 🤔 

But an inventory item would make it lootable, which would defeat the purpose of limiting it. 😅

**Comment by Zarrck** [comment]
Limiting to one per player would also mean three per tank meaning one downed crew could still take out most of a platoon in the city

Maybe Limiting to one per tank? But that has downsides as well. Players would have to be fast enough to get the charge on bailing among others

The reason for the charge to exist in the first place is - as i understand it - to allow forcing the total destruction of an enemy asset (and maybe to force more focus on SA by commanders while in cities?). Maybe there is another solution for this problem?

**Comment by nomisum** [comment]
we could limit this to empty vehicles but that means there would be no "last resort tactic" for bailed crew. which could be fine. they could get a mine to place instead. that could still have an impact but would not encourage CQB.

**Comment by DerZade** [comment]
> The reason for the charge to exist in the first place is - as i understand it - to allow forcing the total destruction of an enemy asset (and maybe to force more focus on SA by commanders while in cities?). Maybe there is another solution for this problem?

This is important. In a perfect world you wouldn't use the explosives to "attack" but only to destroy a already disabled tank. 

How about only allowing to attach explosives to an already disabled tank. Shouldn't be that hard to figure out if a tank is disabled 🤔 
This would mean that a tank crew wouldn't be able to defend themselves. Which I would say is favorable 🤔, because it discourages infantry rambo behavior 😅, but if you disagree, we could add a single LAT to the inventory of each tank as a trade off. (i. e. there is a non zeroable M72 LAW included in GM)

**Comment by nomisum** [comment]
> already disabled tank. Shouldn't be that hard to figure out if a tank is disabled 🤔

what do you mean by disabled exactly? cant move? combat disabled?

**Comment by Zarrck** [comment]
Unoccupied should be sufficient

**Comment by DerZade** [comment]
> what do you mean by disabled exactly? cant move? combat disabled?

disabled = can't neither move nor shoot (at least in my opinion) 


> Unoccupied should be sufficient

I disagree. That would mean that if the the tank is just not able to move and shoot, but the crew survived, the crew could just stay inside the tank and block the explosive charge until backup arrives

**Comment by nomisum** [comment]
bumped range from 4 to 7 m in hope to counteract
